src/OpenTelemetry.Abstractions/Trace/Export/ISpanData.cs

```diff
@@ -44,47 +44,47 @@ public interface ISpanData
         string Name { get; }
 
         /// <summary>
-        /// Gets the start timestamp.
+        /// Gets the start <see cref="Timestamp"/>.
         /// </summary>
         Timestamp StartTimestamp { get; }
 
         /// <summary>
-        /// Gets the collection of attributes.
+        /// Gets the collection of <see cref="IAttributes"/> objects.
         /// </summary>
         IAttributes Attributes { get; }
 
         /// <summary>
-        /// Gets the collection of annotations.
+        /// Gets the collection of <see cref="ITimedEvents{IEvent}"/> objects.
         /// </summary>
-        ITimedEvents<IAnnotation> Annotations { get; }
+        ITimedEvents<IEvent> Events { get; }
 
         /// <summary>
-        /// Gets the collection of message events.
+        /// Gets the collection of <see cref="ITimedEvents{IMessageEvent}"/> objects.
         /// </summary>
         ITimedEvents<IMessageEvent> MessageEvents { get; }
```

Comment:
I thought you are removing it. I'm confused =)

Reply:
From the suggestion last night I was planning to create a follow up PR for removing MessageEvents to address #16 and keep this PR fucused on #9 . I'll push a change to revert the ZipKin changes soon.

Reply:
Okay, reverted Zipkin changes and will plan to work on #16 after this PR is complete.